doc/whats_new/v0.20.rst

```diff
@@ -927,7 +938,7 @@ These changes mostly affect library developers.
   ``check_set_params`` test which checks that ``set_params`` is equivalent to
   passing parameters in ``__init__`` and warns if it encounters parameter
   validation. :issue:`7738` by :user:`Alvin Chiang <absolutelyNoWarranty>`
-  
+
```

Comment:
I agree this is an improvement, but the issue is that it can create merge conflicts in other PRs. I would suggest you disable this feature in your editor for your future PRs.